sites/perioimplantadvisory/server/templates/subscribe/index.marko

```diff
@@ -30,10 +30,8 @@ $ const newsletterDescription = 'Stay up-to-date on industry news and events, ne
           <p>${newsletterDescription}</p>
           <p><strong><a href="/subscribe/email">Subscribe / Manage Preferences</a></strong></p>
           <hr>
```

Comment:
I would remove the `<hr>` tag now that there’s nothing below

Reply:
done